worker/worker.go

```diff
@@ -11,12 +11,14 @@ import (
 	"github.com/moby/buildkit/frontend"
 	"github.com/moby/buildkit/solver"
 	digest "github.com/opencontainers/go-digest"
+	specs "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
 type Worker interface {
 	// ID needs to be unique in the cluster
 	ID() string
 	Labels() map[string]string
+	Platforms() []specs.Platform
```

Comment:
What should we do with?
```
38 	LabelOS          = labelPrefix + "os"          // GOOS
39 	LabelArch        = labelPrefix + "arch"        // GOARCH
```

Reply:
Currently, they could be still used for constraints with filters but as they are just `GOOS/GOARCH` they may not be that useful. We can reevaluate if this is more complete. The `Platform` field is more like a "target platform" for vertex and less like a value based constraint.

Reply:
Maybe better to remove these labels for now?

Reply:
Ok, I'll remove

Reply:
LGTM after removal of `LabelOS` and `LabelArch` consts

Reply:
done